src/harness/fourslash.ts

```diff
@@ -1941,6 +1941,33 @@ module FourSlash {
             }
         }
 
+        public verifyDocCommentTemplate(expected?: ts.TextInsertion) {
+            const name = "verifyDocCommentTemplate";
+            let actual = this.languageService.getDocCommentTemplateAtPosition(this.activeFile.fileName, this.currentCaretPosition);
+
+            if (expected === undefined) {
+                if (actual === undefined) {
+                    return;
+                }
+                else {
+                    this.raiseError(name + ' failed - expected no template but got {newText: \"' + actual.newText + '\" caretOffset: ' + actual.caretOffset + '}');
+                }

```

Comment:
The only issue is that we don't want to perform the remaining checks if both `expected === undefined` and `actual === undefined` (we'll get null-dereference errors).